pyseir/cli.py

```diff
@@ -206,12 +236,13 @@ def generate_whitelist():
 @click.option(
     "--state", help="State to generate files for. If no state is given, all states are computed."
 )
-@click.option("--states-only", default=False, is_flag=True, type=bool, help="Only model states")
+@click.option("--states-only", default=False, is_flag=True, type=bool, help="Deprecated")
```

Comment:
:( - fwiw i use this a lot and find it pretty helpful!

Reply:
It's deprecated in the sense that this flag never actually changed anything. Calling run_infer_rt from the CLI never propagated down to running counties for the Infection Rate calculations. Right now build_all calls the cli method for states, but calls the direct method for fips. So calling it with the flag is the same as calling it without the flag.

Reply:
I changed the copy in the text to reflect that the flag is currently ignored and always defaults to states-only right now.